python/src/scipp/plot/tools.py

```diff
@@ -109,14 +94,45 @@ def make_fake_coord(dim, size, unit=None):
     return sc.Variable(dims=[dim], **args)
 
 
-def _find_min_max(array, params):
-    if params["vmin"] is None or params["vmax"] is None:
-        if params["log"]:
-            with np.errstate(divide="ignore", invalid="ignore"):
-                valid = np.ma.log10(array)
-        else:
-            valid = np.ma.masked_invalid(array, copy=False)
-    if params["vmin"] is None:
-        params["vmin"] = valid.min()
-    if params["vmax"] is None:
-        params["vmax"] = valid.max()
+def vars_to_err(v):
+    with np.errstate(invalid="ignore"):
+        v = np.sqrt(v)
+    non_finites = np.where(np.logical_not(np.isfinite(v)))
+    v[non_finites] = 0.0
```

Comment:
Can https://numpy.org/doc/stable/reference/generated/numpy.nan_to_num.html do the work?